digital_asset_types/src/dapi/candy_machines_by_size.rs

```diff
@@ -0,0 +1,247 @@
+use crate::dao::prelude::{CandyGuard, CandyGuardGroup, CandyMachineData};
+use crate::dao::{
+    candy_guard, candy_guard_group, candy_machine, candy_machine_creators, candy_machine_data,
+};
+use crate::rpc::filter::CandyMachineSorting;
+use crate::rpc::response::CandyMachineList;
+use crate::rpc::{
+    CandyGuard as RpcCandyGuard, CandyGuardData, CandyGuardGroup as RpcCandyGuardGroup,
+    CandyMachine as RpcCandyMachine, CandyMachineData as RpcCandyMachineData,
+};
+use sea_orm::{query::*, DbErr};
+use sea_orm::{ColumnTrait, DatabaseConnection, EntityTrait, ModelTrait};
+
+use super::candy_machine::{to_creators, transform_optional_pubkeys};
+use super::candy_machine_helpers::{
+    get_candy_guard_group, get_candy_machine_data, get_freeze_info,
+};
+
+pub async fn get_candy_machines_by_size(
```

Comment:
This looks pretty similar to `get_candy_machines_by_creator`.  What do you think about combining/parameterizing some of the query functionality into a common function or functions?  For example could you pass in a `Condition that is either about the creator expression or about the size?

Reply:
Could you create a Linear task for this and assign to Austin?

Reply:
P-689 created